### ISSUE TITLE
the mobile title in many articles is a date instead of a shortened title, which is confusing and unhelpful

### ISSUE THREAD

**Issue by Protonull** [ziglang/www.ziglang.org#273]
When reading the recent news article about 0.11.0 being postponed, I noticed that the article didn't have a date. I took a quick look at the page's source to see if it was included as a meta tag, but noticed that it was included within a span that's only shown on mobile. I did a quick window-size test and yup, the date of the article is only shown when the browser presumes you're on mobile.

![Screenshot from 2023-07-20 03-13-37](https://github.com/ziglang/www.ziglang.org/assets/52572989/3d5a8847-1860-4e24-a482-586ab82a8a8f)

![Screenshot from 2023-07-20 03-13-47](https://github.com/ziglang/www.ziglang.org/assets/52572989/2a34817b-8e7f-4ae4-a419-872904780b67)

**Comment by Protonull** [comment]
This seems to have been "fixed" by also removing the date from the "mobile" version.

**Comment by Protonull** [comment]
> why did you close this? that only means both are broken now

I mean, it's been well over a year and the two layouts are now consistent with each other. I can only assume that the choice to remove the date from the 'mobile' version instead of adding it to the 'desktop' version was a deliberate choice. I'd have preferred the former but c'est la vie. I'm sorry, I just thought this issue was moot.

**Comment by nektro** [comment]
the news articles have dates in their metadata and the backend recently switched out for a new static site generator. i think its quite likely it was an oversight

**Comment by andrewrk** [comment]
it's not a publication date, it's the "mobile title", and somebody just picked a really poor mobile tile for that particular news article. instead it should be a shorter rewording of the longer title, I guess. Idk, I wouldn't have made there be a different title for mobile in the first place.

**Comment by nektro** [comment]
that's not what this issue was trying to communicate, the previous title was correct.
take https://ziglang.org/news/migrate-to-self-hosting/ for example
https://github.com/ziglang/www.ziglang.org/blob/main/content/en-US/news/migrate-to-self-hosting/index.smd#L3
`2024-09-12` or `12 Sep 2024` is not displayed anywhere in the document

**Comment by nektro** [comment]
https://github.com/ziglang/www.ziglang.org/blob/main/content/en-US/news/0.11.0-postponed-again.smd#L10
it seems the mobile title on that particular article was indeed made the date but thats more an unfortunate coincidence than the root cause here

**Comment by Protonull** [comment]
> that's not what this issue was trying to communicate, the previous title was correct

As you say, it was about the lack of a date on the 'desktop' version. It did seem a little odd to me that the 'mobile' version had the date in the grey bar, but I checked some other articles which also had the date there and figured it was a deliberate design choice. Turns out I must've lucked out because the other articles also had similarly "poor" mobile titles. It would be nice to have the date within the article itself (as well as meta-tags) to not then have to cross-reference articles with the news-list page, which was the original intent behind this issue.
